### PR TITLE
Fixed lints in some of the packages in pkg/

### PR DESCRIPTION
**Pull request by Raviv-S**
Signed-off-by: Raviv Schaffer <raviv.schaffer@ibm.com>

### CODE REVIEW

File: pkg/multicluster/local/local_cluster_manager.go [hunk 1]
```diff
@@ -106,9 +106,9 @@ func (cm *localClusterManager) DeleteBlueprint(cluster string, namespace string,
 }
 
 // NewClusterManager creates an instance of ClusterManager for a local cluster configuration
-func NewClusterManager(client client.Client, namespace string) (multicluster.ClusterManager, error) {
+func NewClusterManager(clnt client.Client, namespace string) (multicluster.ClusterManager, error) {
```
Comment (shlomitk1) [inline pkg/multicluster/local/local_cluster_manager.go:109]: Use cl for client as we do in some other files

File: pkg/storage/storage_provisioning.go [hunk 2]
```diff
@@ -97,16 +102,16 @@ func getValue(obj map[string]interface{}, path ...string) string {
 
 func equal(required *ProvisionedBucket, existing *unstructured.Unstructured) bool {
 	obj := existing.UnstructuredContent()
-	if required.Name != getValue(obj, "spec", "local", "bucket") {
+	if required.Name != getValue(obj, specLiteral, localLiteral, "bucket") {
```
Comment (shlomitk1) [inline pkg/storage/storage_provisioning.go:105]: "spec" and "local" are replaced with constants but no complaints for "bucket"?

Comment (Raviv-S) [reply pkg/storage/storage_provisioning.go:105]: The complaints are only if there are more than 4 instances of the string.
"bucket" has only 2 instances.

File: pkg/multicluster/local/local_cluster_manager.go (hunk 1 above)
Comment (Raviv-S) [reply pkg/multicluster/local/local_cluster_manager.go:109]: I'll change it

File: pkg/storage/storage_provisioning.go (hunk 2 above)
Comment (roytman) [reply pkg/storage/storage_provisioning.go:105]: @Raviv-S even if the linter doesn't complain about `bucket ` and it has only 2 instances, I'd define a constant for it too.

File: pkg/multicluster/razee/razee_cluster_manager.go [hunk 3]
```diff
@@ -70,16 +73,16 @@ func (r *razeeClusterManager) GetClusters() ([]multicluster.Cluster, error) {
...
 		}
 		// If no content for the resource was found the cluster is not part of Fybrik or not installed
 		// correctly. Fybrik should ignore those clusters and continue.
 		if resourceContent == nil {
-			r.log.Error().Err(err).Str("cluster", c.Name).Msg("Resource content returned is nil! Skipping cluster")
+			r.log.Error().Err(err).Str(clusterKey, c.Name).Msg("Resource content returned is nil! Skipping cluster")
 			continue
 		}
-		scheme := runtime.NewScheme()
+		scheme = runtime.NewScheme()
```
Comment (Raviv-S) [inline pkg/multicluster/razee/razee_cluster_manager.go:85]: @roytman 
I'm not sure if the re-assignment of `scheme` would  create a problem here

**Pull request merged**